### PR TITLE
Add load_and_transform_thermal_data

### PR DESCRIPTION
**Pull request by Abhishekmallik**
Adding the load_and_transform_thermal_data function to data.py

### CODE REVIEW

**Comment by zhang-ziang** [comment]
Thanks for the code!
I find something maybe wrong.
As the thermal data is not the normal image, I think the transforms.Normalize() in transforms.Compose[] should be removed.
I'm recurrencing the metrics on LLVIP dataset, and I find that when the Normalize is added, the binary classification accuracy is about 55.2 and when removed, acc is  about 62.4. The latter is more consistent with the paper of ImageBind.

**Comment by Jade999** [comment]
> Thanks for the code! I find something maybe wrong. As the thermal data is not the normal image, I think the transforms.Normalize() in transforms.Compose[] should be removed. I'm recurrencing the metrics on LLVIP dataset, and I find that when the Normalize is added, the binary classification accuracy is about 55.2 and when removed, acc is about 62.4. The latter is more consistent with the paper of ImageBind.

@zhang-zian hello, you mean we need to delete transforms.Normalize operation in transforms.Compose，may I ask if this has been confirmed?

**Comment by zhang-ziang** [comment]
> > Thanks for the code! I find something maybe wrong. As the thermal data is not the normal image, I think the transforms.Normalize() in transforms.Compose[] should be removed. I'm recurrencing the metrics on LLVIP dataset, and I find that when the Normalize is added, the binary classification accuracy is about 55.2 and when removed, acc is about 62.4. The latter is more consistent with the paper of ImageBind.
> 
> @ZHANG-ZIAN hello, you mean we need to delete transforms.Normalize operation in transforms.Compose，may I ask if this has been confirmed?

Maybe. I'm not sure because when I use this method to preprocess the LLVIP test set, the thermal-image retrieval performance is not good as I expect, as ImageBind is train on thermal-image pairs. I also retrained the thermal trunk follow ImageBind paper, but still can't get very good results. 
I also get bad result on depth modality. You can find some discussions in the issue where everyone didn't get good results.
Perhaps it is best to wait for the authors themselves to release these preprocessing function code.

**Comment by Jade999** [comment]
Okay, thank you very much for your experience. Let's wait for the official code
@facebook-github-bot @Abhishekmallik

**Comment by Parv-Maheshwari** [comment]
Hi @Abhishekmallik  @rohitgirdhar can you please provide an update on the dataloader for thermal.
